mapper/orm/src/main/java/org/hibernate/search/mapper/orm/reporting/impl/HibernateOrmMappingHints.java

```diff
@@ -22,4 +22,7 @@ public interface HibernateOrmMappingHints extends BackendMappingHints {
 	@Message(value = "This is likely a bug, as Hibernate ORM entities should always be loadable from the database.")
 	String noEntityProjectionAvailable();
 
+	@Override
+	@Message( "Add the @ScaledNumberField annotation to this property and configure the `decimalScale` attribute as necessary." )
```

Comment:
I think the exception can also arise when using `TypeBinder`/`PropertyBinder`, so it's entirely possible no `@*Field` annotation is involved... Maybe some less definitive message would be better:

```suggestion
	@Message( "If you used a @*Field annotation here, make sure to use @ScaledNumberField and configure the `decimalScale` attribute as necessary." )
```

Same for the standalone POJO Mapper.